fdbclient/StatusClient.actor.cpp

```diff
@@ -462,13 +462,15 @@ ACTOR Future<StatusObject> statusFetcherImpl( Reference<ClusterConnectionFile> f
 	// This could be read from the JSON but doing so safely is ugly so using a real var.
 	state bool quorum_reachable = false;
 	state int coordinatorsFaultTolerance = 0;
-	state Reference<AsyncVar<Optional<ClusterInterface>>> clusterInterface(new AsyncVar<Optional<ClusterInterface>>);
+
+	if(!clusterInterface) {
```

Comment:
Does this `if` block ever get triggered? It looks like the only caller of it wouldn't pass in an unset `Reference`.

Reply:
I think this was a relic of a previous implementation, so I am removing it